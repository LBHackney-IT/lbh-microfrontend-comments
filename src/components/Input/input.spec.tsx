import React from 'react';
import { render } from '@testing-library/react';

import { Input } from '@components';

test('it renders correctly', () => {
    const { container } = render(<Input />);
    expect(container).toMatchSnapshot();
});

test('it renders correctly with an error', () => {
    const { container } = render(<Input error />);
    expect(container).toMatchSnapshot();
});
