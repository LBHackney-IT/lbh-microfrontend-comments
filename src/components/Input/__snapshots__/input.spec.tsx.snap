// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<div>
  <input
    class="govuk-input lbh-input"
  />
</div>
`;

exports[`it renders correctly with an error 1`] = `
<div>
  <input
    class="govuk-input lbh-input govuk-input--error"
  />
</div>
`;
